lib/ansible/executor/task_executor.py

```diff
@@ -535,7 +535,7 @@ def _execute(self, variables=None):
             # update the local copy of vars with the registered value, if specified,
             # or any facts which may have been generated by the module execution
             if self._task.register:
-                vars_copy[self._task.register] = wrap_var(result.copy())
+                vars_copy[self._task.register] = wrap_var(result)
```

Comment:
Was the previous result.copy() necessarily?  I don't know what the intention of the original result.copy() was so I'm not sure if it can be removed.

Reply:
It was removed at the request of bcoca in chat as not being needed.  We actually have other lines that look just like this, that didn't have `.copy` in the same method.